src/main/resources/db/changelog/alert/5.0.0/changelog-remove-hipchat.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+    <changeSet author="jrichard" id="1560948151336-1">
+       <delete schemaName="ALERT"
+               tableName="DESCRIPTOR_CONFIGS">
+            <where>ID IN
+                    (SELECT CONFIG_ID FROM ALERT.CONFIG_GROUPS WHERE JOB_ID IN
+                        (SELECT JOB_ID FROM ALERT.CONFIG_GROUPS WHERE CONFIG_ID IN
+                            (SELECT CONFIG_ID FROM ALERT.FIELD_VALUES WHERE FIELD_VALUE = 'channel_hipchat')
```

Comment:
Does this delete both jobs and global?

Reply:
This query is to ensure the HipChat jobs are entirely deleted. The cascading deletions take care of the HipChat config in the Job but not the Provider config in the job.
The global config is removed via the cascading deletion, triggered by descriptor being deleted and the registerd fields being deleted.